src/jobs/pages/job-offers-admin.component.vue

```diff
@@ -62,24 +65,49 @@ import Dialog from "primevue/dialog";
               ><i :class="PrimeIcons.SEARCH" />
               <InputText
                 v-model="filters['global'].value"
-                placeholder="Search..." />
+                placeholder="🔍 Search..." />
             </span>
           </div>
         </template>
+        <template #empty> No offers found. </template>
+        <template #loading> Loading offers data. Please wait. </template>
 
-        <Column selection-mode="multiple" :exportable="false" class="w-12">
+        <Column
+          selection-mode="multiple"
+          header-style="width: 3em"
```

Comment:
Don't use inline styles.

```suggestion
          header-class="w-12"
```